Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN pip install --upgrade pip
 RUN python -m pip install catkin_tools casadi utm xmltodict pygame matplotlib==2.2.5 simple_pid tornado pymongo bandit
 RUN pip3 install --upgrade pip
 RUN python3 -m pip install geopy pyyaml rospkg utm psycopg2 bandit
+RUN pip2.7 install psycopg2
```

Comment:
Can we add this to here instead? https://github.com/unicorns/ros-dev-docker/pull/4/files#diff-3254677a7917c6c01f55212f86c57fbfR21

Reply:
pretty sure `python -m pip` is equivalent to `pip2.7` or `pip`

Reply:
Yup, totally, wasn't sure of that